files/image_config/ntp/mgmtvrf_ntpq

```diff
@@ -0,0 +1,5 @@
+ip link set dev lo master mgmt
+sleep 1
+ntpq $@
+sleep 1
+ip link set dev lo nomaster
```

Comment:
why the ntpq depends on lo in the mgmt vrf? can we create another lo0 and attached the lo0 to mgmt vrf? what exactly is needed here to support ntpq?

Reply:
it is not intuitive that we need to move the lo back-and-forth to the mgmt vrf to start ntpq. if we need a lo in the mgmt vrf, can we just create one there and use it.

Reply:
Yes, we shall create a new "lo0" (or "lo1") interface and use it for this purpose.

Reply:
@lguohan: OK. We will create lo-m, test it and keep you posted.

Reply:
Created lo-m  in interfaces.j2 and tested.

Verified NTP using “ntpq -p” and “ntpstat” before enabling management VRF.
Enabled management vrf using “config vrf add mgmt” command that is explained in PR Azure/sonic-utilities#463
Verified NTP in management VRF using “cgexec” as follows.
root@sonic:/home/admin# date
Fri Aug 30 15:31:53 UTC 2019
root@sonic:/home/admin# cgexec -g l3mdev:mgmt ntpq -p
remote refid st t when poll reach delay offset jitter
23.92.29.245 .STEP. 16 u - 64 0 0.000 0.000 0.000
*204.2.134.164 44.24.199.34 3 u 9 64 1 200.601 -0.199 2.589

root@sonic:/home/admin# cgexec -g l3mdev:mgmt ntpstat
synchronised to NTP server (204.2.134.164) at stratum 4
time correct to within 1085 ms
polling server every 64 s
root@sonic:/home/admin# ps -ax | grep ntp
9761 ? Ssl 0:00 /usr/sbin/ntpd -p /var/run/ntpd.pid -g -u 106:110
9799 ttyS1 S+ 0:00 grep ntp
root@sonic:/home/admin#